tf-modules/cumulus-message-adapter/README.md

```diff
@@ -0,0 +1,20 @@
+# Cumulus Message Adapter
+
+This pacakge creates a lambda layer for upload/use of the [Cumulus Message Adapter](https://github.com/nasa/cumulus-message-adapter).    This can be utilized in combination with the kes deployment tool per the [input/output Cumulus documentation](http://localhost:3000/cumulus/docs/next/workflows/input_output).
```

Comment:
The docs link here should point to our public docs, not `localhost:3000`

Reply:
🔔  Good catch. bd867a74d